webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/pdf/GeneralScrambleSheet.kt

```diff
@@ -80,43 +74,20 @@ class GeneralScrambleSheet(scrambleSet: ScrambleSet, activityCode: ActivityCode)
         }
     }
 
-    private fun getFontConfiguration(scrambleColumnWidth: Float, scrambleImageSize: Float, scrambles: List<String>): Font {
-        val availableScrambleHeight = scrambleImageSize - 2 * SCRAMBLE_IMAGE_PADDING
-
-        val availableArea = Rectangle(scrambleColumnWidth - 2 * SCRAMBLE_PADDING_HORIZONTAL,
-            (availableScrambleHeight - SCRAMBLE_PADDING_VERTICAL_TOP - SCRAMBLE_PADDING_VERTICAL_BOTTOM))
-
-        val longestScramble = scrambles.maxBy { it.length } ?: ""
-
-        val longestAlignedScramble = scrambles.map { StringUtil.padTurnsUniformly(it, WIDEST_CHAR_STRING) }
-            .maxBy { it.length } ?: longestScramble
+    private fun getFontConfiguration(availableArea: Rectangle, scrambles: List<String>): Font {
+        val longestScramble = scrambles.flatMap { it.split(NEW_LINE) }.maxBy { it.length } ?: ""
```

Comment:
There is `String?.orEmpty()` in Kotlin. So this can be `maxBy { it.length }.orEmpty()`

Reply:
Nice! I kept line 89 standard though.

Reply:
Fixed